docs/guides/classical-feedforward-and-control-flow.ipynb

```diff
@@ -202,196 +193,112 @@
+       "<Image src=\"/docs/images/guides/classical-feedforward-and-control-flow/extracted-outputs/3bdb923d-c466-4743-99b1-d46fda9660ef-0.avif\" alt=\"Output of the previous code cell\" />"
       ]
      },
-     "execution_count": 6,
+     "execution_count": 23,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "qubits = QuantumRegister(1)\n",
-    "clbits = ClassicalRegister(1)\n",
-    "circuit = QuantumCircuit(qubits, clbits)\n",
-    "(q0,) = qubits\n",
-    "(c0,) = clbits\n",
-    "\n",
-    "with circuit.for_loop(range(5)) as _:\n",
-    "    circuit.x(q0)\n",
-    "circuit.measure(q0, c0)\n",
-    "\n",
-    "circuit.draw(\"mpl\")\n",
-    "\n",
-    "# example output counts: {'1': 1024}"
+    "qc.draw(output=\"mpl\", style=\"iqp\", idle_wires=False, fold=-1)"
    ]
   },
   {
    "cell_type": "markdown",
-   "id": "6f1050b0-58a5-46e1-95c6-980db3f09c00",
+   "id": "0378a45f-7a28-48bc-993d-0e4308d38bfc",
    "metadata": {},
    "source": [
-    "## While loop\n",
+    "## Limitations\n",
     "\n",
-    "A while loop is used to repeat instructions while some condition is satisfied.\n",
-    "\n",
-    "In the example below, we apply Hadamard gates to two qubits and measure them. Then, we create a while loop that repeats this procedure while the measurement outcome is 11. As a result, the final measurement should never be 11, with the remaining possibilities appearing with approximately equal frequency."
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 7,
-   "id": "95795aa7-9d4d-4c35-ba0e-48e6ba4f41b9",
-   "metadata": {},
-   "outputs": [
-    {
-     "data": {
-      "text/plain": [
-       "<Image src=\"/docs/images/guides/classical-feedforward-and-control-flow/extracted-outputs/95795aa7-9d4d-4c35-ba0e-48e6ba4f41b9-0.svg\" alt=\"Output of the previous code cell\" />"
-      ]
-     },
-     "execution_count": 7,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "qubits = QuantumRegister(2)\n",
-    "clbits = ClassicalRegister(2)\n",
-    "circuit = QuantumCircuit(qubits, clbits)\n",
-    "\n",
-    "q0, q1 = qubits\n",
-    "c0, c1 = clbits\n",
-    "\n",
-    "circuit.h([q0, q1])\n",
-    "circuit.measure(q0, c0)\n",
-    "circuit.measure(q1, c1)\n",
-    "with circuit.while_loop((clbits, 0b11)):\n",
-    "    circuit.h([q0, q1])\n",
-    "    circuit.measure(q0, c0)\n",
-    "    circuit.measure(q1, c1)\n",
-    "\n",
-    "circuit.draw(\"mpl\")\n",
+    "Be aware of the following constraints when using dynamic circuits:\n",
     "\n",
-    "# example output counts: {'01': 334, '10': 368, '00': 322}"
+    "- Whenever you do a measure followed by an `if (c)`, the values that are measured and subsequently used in the if condition has a limit of **60 bits**.\n",
+    "- Nested conditionals are not allowed.\n",
+    "- Having `reset` inside conditionals is not supported."
    ]
```

Comment:
We should add a note saying that we will continue to rollout new features and improvements for dynamic circuits through the year, and some of these limitations will be lifted.